.github/workflows/reusable-python-tests.yml

```diff
@@ -31,6 +31,14 @@ jobs:
       FORCE_COLOR: 3
       GITHUB_TOKEN: ${{ github.token }}
     steps:
+      # free up some space on the Linux runners
+      # the nox environments can get quite large, especially if torch is installed
+      - name: Free up space
+        if: runner.os == 'Linux'
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
I think in the other PR, I decided to remove some other directories from the runners. I think I avoided to remove the AGENT one because I thought that one might actually be semi-useful.
Would you mind checking?